src/app/quiz/components/checkboxChoiceInput.js

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import Checkbox from '@mui/material/Checkbox';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import Grid from "@mui/material/Grid";
+import Button from "@mui/material/Button";
+import {SvgIcon} from "@mui/material";
+import Container from "@mui/material/Container";
+import Box from "@mui/material/Box";
+import useSWR from "swr"
+import { useState } from 'react';
+
+const post_fetcher = (...args) => fetch(...args).then(res => res.json())
+
+function ChoiceVariantButton({ choice, question }) {
+
+  const [bgColor, setBgColor] = useState("#FAF5EF");
+
+  function handleClick(e, data) {
+    // check checkbox status
+    setBgColor("#D7AF7F")
+  }
+
+  return (
+    <FormControlLabel
+      value={choice.id}
+      onClick={handleClick}
+      control={
+        <Checkbox sx={{ display: "none" }} />
+      }
+      label={choice.text}
+      labelPlacement="end"
+      sx={{
+        backgroundColor: bgColor,
+        padding: 3,
+        borderRadius: 1,
+        textAlign: "center",
+        border: "none",
+        width: "100%",
+        mx: 0,
+
+        borderRadius: "5px",
+        height: 118,
+        flexShrink: 0,
+      }}
+
+      componentsProps={{
+        typography: {
+          sx: {
+            color: "#08202F",
+            align: "center",
+            fontFamily: "Lora",
+            fontSize: 20,
+            fontStyle: "normal",
+            fontWeight: 500,
+            lineHeight: "normal",
+
+            // ml: 2,
+          }
+        }
+      }}
+    />
+  )
+}
+
+
+export default function CheckboxChoiceInput({question, handleAnswerSubmit, postAnswerIsMutating}) {
+  function handleSubmit(value) {
+    handleAnswerSubmit({choice: value})
+  function handleSubmit(event, data) {
+    console.log(event)
+    console.log(data)
+    const formData = new FormData(event.currentTarget);
+    const selectedOptions = formData.getAll("options");
+    console.log(selectedOptions)
+
+    handleAnswerSubmit({choice: data})
+    // Using FormData to collect checkbox values
+  }
+
+  return (
+    <FormControl sx={{mt: 5}} onSubmit={handleSubmit}>
```

Comment:
What for do we need FormControl and FormGroup here?
FormControl provides context, which we do not use here.
Form Group provides layout, which we also do not use here.